src/logger/index.js

```diff
@@ -74,8 +54,30 @@ const request = req => systemLogger.request({
 	timestamp: new Date(),
 });
 
+const logger = winston.createLogger({
+	level: logLevel,
+	levels: winston.config.syslog.levels,
+	format: winston.format.combine(
+		winston.format.timestamp(), // adds current timestamp
+		//	winston.format.ms(),	// adds time since last log
+		winston.format.prettyPrint(), // Use 'winston.format.prettyPrint()' for string output
+		// todo format for MethodNotAllowed: Method `update` is not supported by this endpoint.  -> MethodNotAllowed:
+	),
+	transports: [
```

Comment:
why not adding a sentry transport here, because of sentry using the sentry console integration?

Reply:
It need an additional package. I can use https://www.npmjs.com/package/winston-sentry-raven-transport and remove sentry/integrations. I think adding by winston is very nice, but the packages are very low downloads per week.

Reply:
Nearly 2000 downloads per week should be ok!? I also would prefer to log directly via winston transporter